Need help to load the plugin.

From  the version V0.8.28, I could not run the embulk with plugin. It seems that the code about setup  the embulk(embulk-cli) has modified a lot.

embulk run -L path/to/myplugin config.yml
went wrong.

chronos@localhost ~/Downloads/data/test_multi $ embulk run -L ~/Downloads/embulk-input-filename -L ~/Downloads/embulk-output-joinfile seed2.yml 
2017-08-10 18:10:45.855 +0200: Embulk v0.8.28
SyntaxError: <script>:1: syntax error, unexpected end-of-file
$LOAD_PATH.unshift File.expand_path(File.join(__internal_plugin_path__, 'lib')
Exception in thread "main" org.jruby.embed.ParseFailedException: (SyntaxError) <script>:1: syntax error, unexpected end-of-file
$LOAD_PATH.unshift File.expand_path(File.join(__internal_plugin_path__, 'lib')
        at org.jruby.embed.internal.EmbedRubyRuntimeAdapterImpl.runParser(EmbedRubyRuntimeAdapterImpl.java:205)
        at org.jruby.embed.internal.EmbedRubyRuntimeAdapterImpl.parse(EmbedRubyRuntimeAdapterImpl.java:94)
        at org.jruby.embed.ScriptingContainer.parse(ScriptingContainer.java:1239)
        at org.jruby.embed.ScriptingContainer.runScriptlet(ScriptingContainer.java:1299)
        at org.embulk.cli.EmbulkRun.setupLoadPaths(EmbulkRun.java:719)
        at org.embulk.cli.EmbulkRun.runSubcommand(EmbulkRun.java:449)
        at org.embulk.cli.EmbulkRun.run(EmbulkRun.java:98)
        at org.embulk.cli.EmbulkBundle.checkBundleWithEmbulkVersion(EmbulkBundle.java:42)
        at org.embulk.cli.EmbulkBundle.checkBundle(EmbulkBundle.java:15)
        at org.embulk.cli.Main.main(Main.java:26)
Caused by: org.jruby.exceptions.RaiseException: (SyntaxError) <script>:1: syntax error, unexpected end-of-file
$LOAD_PATH.unshift File.expand_path(File.join(__internal_plugin_path__, 'lib')

--- a/embulk-core/src/main/java/org/embulk/EmbulkRunner.java
+++ b/embulk-core/src/main/java/org/embulk/EmbulkRunner.java
@@ -563,7 +563,7 @@ public class EmbulkRunner
         // # always has require_paths = ["lib"].
         for (final String pluginPath : pluginPaths) {
             globalJRubyContainer.put("__internal_plugin_path__", pluginPath);
-            globalJRubyContainer.runScriptlet("$LOAD_PATH.unshift File.expand_path(File.join(__internal_plugin_path__, 'lib')");
+            globalJRubyContainer.runScriptlet("$LOAD_PATH.unshift File.expand_path(File.join(__internal_plugin_path__, 'lib'))");
             globalJRubyContainer.remove("__internal_plugin_path__");
         }
 

